### ISSUE TITLE
io.imshow_collection always creating a new figure

### ISSUE THREAD

**Issue by TheShadow29** [scikit-image/scikit-image#2675]
I am using `io.imshow_collection` in ipython notebook. Here is the code
```
%matplotlib
from skimage import io
import matplotlib.pyplot as plt
io.use_plugin('matplotlib')
a1 = io.imread_collection('path_to_img_folder/*')
plt.figure()
io.imshow_collection(a1)
```
This code opens up two figures Figure1 and Figure2, with `io.imshow_collection(a1)` showing the images on the second figure. Instead if I were to use only a single image, then only one figure shows up, with the image being shown. Below is the code for one img.
```
%matplotlib
from skimage import io
import matplotlib.pyplot as plt
io.use_plugin('matplotlib')
a1 = io.imread('path_to_img_folder/img_file')
plt.figure()
io.imshow(a1)
```
Where am I going wrong?

**Comment by soupault** [comment]
You are correct: this line https://github.com/scikit-image/scikit-image/blob/master/skimage/io/_plugins/matplotlib_plugin.py#L156 allows to reuse the existing figure for `imshow`. I understand that this looks inconsistent from user point of view, but it is not clear how to make `imshow_collection` performing in the same way: it need a figure with the number of `axes` equal a number of images in the collection. If the already existing figure doesn't satisfy this condition, it is complicated to remap it (to the best of my knowledge).

To sum up, I'd classify this as a feature of `imshow` rather than a flaw of `imshow_collection`.

**Comment by TheShadow29** [comment]
@soupault Is there any way to get the plt.figure() out from imshow_collection. Because as is, I couldn't find a way to put a title to imshow_collection. My main motivation for this was to use the current plt.figure() and then easily use the plt.title() and be able to set all other attributes.

**Comment by soupault** [comment]
@TheShadow29 At the moment, not. I think, it would be much easier for you to reimplement the function in the way you find convenient.
